managed-ledger/src/main/java/org/apache/bookkeeper/mledger/util/RangeCache.java

```diff
@@ -113,7 +113,7 @@ public Value get(Key key) {
             try {
                 value.retain();
                 values.add(value);
-            } catch (Throwable t) {
+            } catch (IllegalReferenceCountException e) {
```

Comment:
probably we should log something here.
this case must not happen

Reply:
It's expected to happen and it's fine when it happens. It just indicates the entry is being evicted when we're trying to access it. If the retain succeeds, the operation was successful, otherwise the entry is already gone.